README.md

```diff
@@ -4,130 +4,180 @@
-
-You can find example machine learning models at
-[tensorflow/swift-models](https://github.com/tensorflow/swift-models), or try out a tutorial at
-[tensorflow/swift-tutorials](https://github.com/tensorflow/swift-tutorials).
-
-## Related Projects
+> Swift for TensorFlow is **not** intended to remain a long-term fork of the official 
+> Swift language. New language features will eventually go through the Swift evolution process
+> as part of being considered for being pulled into master.
 
 ### Jupyter Notebook support
 
-[Jupyter Notebook](http://jupyter.org/) support for Swift for TensorFlow is under development at
+[Jupyter Notebook](http://jupyter.org/) support for Swift is under development at
 [google/swift-jupyter](https://github.com/google/swift-jupyter).
 
 ## Community
 
-Discussion about Swift for TensorFlow happens on the
+Swift for TensorFlow discussions happen on the
 [swift@tensorflow.org mailing list](https://groups.google.com/a/tensorflow.org/d/forum/swift).
 
-## Bugs Reports and Feature Requests
+For the most part, the core team's development is tracked in
+[JIRA].
 
-Before reporting an issue, please check the [Frequently Asked Questions](FAQ.md) to see if your question has already been addressed.
+### Bugs Reports and Feature Requests
 
-For questions about general use or feature requests, please report them in this repository or send an email to the [mailing list](mailto:swift@tensorflow.org).
+Before reporting an issue, please check the [Frequently Asked Questions](FAQ.md)
+to see if your question has already been addressed.
 
-For Swift compiler bugs introduced by Swift for TensorFlow, please file them to [bugs.swift.org](https://bugs.swift.org) within the “Swift for TensorFlow” component.
+For questions about general use or feature requests, please send an email to
+the [mailing list](mailto:swift@tensorflow.org) or search for relevant issues
+in the [JIRA issue tracker](https://bugs.swift.org/projects/TF/issues/?filter=allopenissues).
 
-For bugs in the example models, please report them to [tensorflow/swift-models](https://github.com/tensorflow/swift-models/issues).
+For the most part, the core team's development is also tracked in
+[JIRA](https://bugs.swift.org/secure/RapidBoard.jspa?rapidView=17&projectKey=TF&view=planning).
 
-## Contributing
+### Contributing
 
 We welcome source code contributions: please read 
 [Contributing Code](https://swift.org/contributing/#contributing-code).
-It is always a good idea to discuss your plans on the mailing list
-before making any major submissions.
+It is always a good idea to discuss your plans on the mailing list before
+making any major submissions.
 
-The compiler and the standard library have the most [open issues](https://github.com/google/swift/issues).
-To get started, you can try to tackle issues labeled "good first issue".
+Check out our [starter bugs](https://bugs.swift.org/issues/?filter=11323) if
+you'd like to get involved but don't know where to start!
```

Comment:
It’s very nice that there’s a bunch of starter bugs to choose from!

Reply:
👍